subscription.go

```diff
@@ -162,19 +162,21 @@ func (r *marathonClient) registerCallbackSubscription() error {
 	return nil
 }
 
-func (r *marathonClient) registerSSESubscription() error {
-	// Prevent multiple SSE subscriptions
+func (r *marathonClient) registerSSESubscriptionOnce() error {
 	if r.subscribedToSSE {
 		return nil
 	}
 
-	request, _, err := r.buildAPIRequest("GET", marathonAPIEventStream, nil)
-	if err != nil {
+	if err := r.registerSSESubscription(); err != nil {
 		return err
 	}
 
-	// Try to connect to stream, reusing the http client settings
-	stream, err := eventsource.SubscribeWith("", r.config.HTTPClient, request)
+	r.subscribedToSSE = true
+	return nil
+}
+
+func (r *marathonClient) registerSSESubscription() error {
```

Comment:
This method was and still is not affected by `RemoveEventsListener()`. The same holds true for the http-server that is opened when you use the `EventsTransportCallback`. This is basically because the SSE connection is the underlying transport and can feed multiple `EventsListener`'s. This is of course actually a desirable behaviour of the software, but I would not consider in this PR and it is, imho, rather a nice to have. Do you think its worth to open a separate issue?